apps/src/code-studio/progress.js

```diff
@@ -175,6 +175,26 @@ progress.renderCourseProgress = function(scriptData) {
 
   const mountPoint = document.createElement('div');
   $('.user-stats-block').prepend(mountPoint);
+
+  const calendarLessonsOverview = scriptData.showCalendar
+    ? (scriptData.lessons || []).map(lesson => {
+        let duration = 0;
+        lesson.activities.forEach(activity => {
+          if (activity.duration) {
+            duration = duration + activity.duration;
+          }
+        });
+        return {
+          id: lesson.id,
+          title: lesson.title,
+          duration,
+          assessment: lesson.assessment,
+          unplugged: lesson.unplugged,
+          url: lesson.levels[0] && lesson.levels[0].url
```

Comment:
I think we want the lessons to link to the lesson plan instead of the first level.